libs/src/services/ethContracts/stakingProxyClient.js

```diff
@@ -20,17 +21,17 @@ class StakingProxyClient extends ContractClient {
 
   async totalStaked () {
     const method = await this.getMethod('totalStaked')
-    return parseInt(await method.call(), 10)
+    return this.toBN(await method.call())
```

Comment:
i think these are all BN already - worth confirming though

Reply:
they're strings, unfortunately. Removing this and doing 
```
console.log(typeof (await token.balanceOf(ownerWallet)))
```
Gives "string"

Reply:
so i did some more digging here and i think our libs code is doing something to strip out the valid types - here's what balanceOf returns from a unit test environment:

```
    let bal = await token.balanceOf(tokenOwnerAddress)
    console.log(web3.utils.isBN(bal))
    console.log(typeof bal)
    assert.equal(bal, INITIAL_SUPPLY)
```

```

  Contract: AudiusToken
true
object
    ✓ Initial token properties + account balances (159ms)


  1 passing (2s)
```

Maybe it's the usage of .call() instead? i noticed that the governance tests use these everywhere (which is not required for a `view` function

Reply:
Took a look here again. The reason this is supported is because `token` in the above is the wrapped method that comes from the `artifacts` library that we shouldn't use in this context. I'm sure there are some valuable abstractions we can explore here, but punting on this for now.